data/coalition/resistance intro.txt

```diff
@@ -0,0 +1,1680 @@
+
+mission "Resistance: Combat Training 2"
+	name "Combat Test"
+	description "Escort Pyakri's fleet to <waypoints>, and help them fight a pirate fleet there to get started with their training. Make sure all of the Lunarium ships survive."
+	landing
+	source "Zug"
+	waypoint "Shaula"
+	to offer
+		has "Resistance: Combat Training 1: done"
+		not "Heliarch Investigation 2: active"
+		not "Heliarch License: done"
+	to fail
+		has "Heliarch Investigation 2: active"
+	on offer
+		conversation
+			`Once you've landed, Pyakri and some other Lunarium members come to your cockpit, where you show them a monitor with the list of ships for sale at the local shipyard. You answer their questions about the ships' abilities, strengths and failings to the best of your knowledge. You also show them the outfits available on <origin>, in case they want to change any of the ship loadouts. After discussing with the others for a few minutes, and going back to check the pages of some ships, Pyakri asks you to help her purchase 1 Bastion, 3 Argosies and 4 Clippers.`
+			choice
+				`	"Are you sure you don't want more Bastions? They're pretty tough, you'd all be safer in them."`
+				`	"So how are you going to actually enter the ships? I thought you wanted to remain secret from humans."`
+					goto secret
+			`	"Much more durable than what we have available, Heliarch ships are. Aware of the danger, everyone is, and to better understand means to survive in more fragile ships, practice with them we must." She answers.`
+				goto help
+			label secret
+			`	"Be outside for long, we cannot, so have the ships brought closer to here, we will need. Wait for nightfall first, though, we should, as easier to sneak quickly into the purchased ships it would be."`
+			label help
+			`	You help her finish the transaction, giving the basic rundown of Southbound's shipyard interface, and ask for the ships to be brought to the hangars beside the <ship>. "A great help you were, Captain. Vital, this will be, if to pass as a human fleet, we are. Be careful in communicating with clients in the area, I must be." Your face showing some confusion, she explains her plans further. "After gotten the gist of fighting human pirates with you, we have, work as bounty hunters, we will. Make enough to pay you back for your services, I intend to. Even if similar to taking jobs in the Coalition, human bounties are, still to communicate minimally with the clients, we may need."`
+			choice
+				`	"Well, I'm sure you'll get the hang of it."`
+					goto accept
+				`	"Are you sure that's a good idea? Since you want to avoid humans noticing you."`
+			`	"What? Oh, worried about the translators, are you? Yes, sound robotic at times, they can, but impede us not, that will. Readied the translating software to work on text messages, we have. Neither seen nor heard, we will be," she touts.`
+			choice
+				`	"I'm more worried that the... 'accent,' will give you away."`
```

Comment:
I wouldn't call it 'accent', maybe peculiar phrasing?

Reply:
I put it in quotes because I couldn't find a quick way to describe it properly, "accent" is more the player character can't really explain what's going on with the Coalition translators in a quick phrase, so "accent" is used loosely to summarize the translators' different speech.
I'm not sure if it's worth changing it to something more wordy, I'll wait on Wizrad for that.

Reply:
Makes sense, but just saying 'accent' seems like it loses the meaning. I wouldn't have understood the accent was referring to the Yoda speech without you comment for clarification, so rewording would probably be a good idea. Maybe "speech patterns"? I'll swing back around to this when I eventually review